openassessment/xblock/staff_area_mixin.py

```diff
@@ -164,7 +166,14 @@ def get_staff_path_and_context(self):
             })
 
         # Include whether or not staff grading step is enabled.
-        context['staff_assessment_required'] = "staff-assessment" in self.assessment_steps
+        staff_assessment_required = "staff-assessment" in self.assessment_steps
```

Comment:
I'm having trouble understanding what is, and what should be going on here. The comment above this block, and the fact that we're just checking for `staff-assessment` in `self.assessment_steps` seem to indicate that we aren't checking for whether the staff step is *required*, rather if it's *enabled*.

However, when verifying on the sandbox, it *does* seem to only show the `{% if staff_assessment_required %}` guarded html when the staff step is actually required.